### ISSUE TITLE
ec2containerservice connection issues

### ISSUE THREAD

**Issue by aleksandersumowski** [boto/boto#3205]
Hi,
I can't get the ec2containerservice connection to work unless I modify boto/ec2containerservice/__init__.py lines 33-34 to:

    from boto.ec2containerservice**.layer1** import EC2ContainerServiceConnection
    return get_regions('**ec2containerservice**', connection_cls=EC2ContainerServiceConnection)

Is it a known issue? Is ec2containerservice support considered ready for prod use?

**Comment by micahhausler** [comment]
Duplicate of #3172

**Comment by aleksandersumowski** [comment]
@micahhausler: not really. The referenced issue is just about missing regions. I mention what seems like incorrect code ... 

* wrong namespace in the line 33
* missing service name when getting regions for the service in the line 34

**Comment by nickrowlandboxuk** [comment]
Would be really good to get this fixed, there's a PR open that fixes this https://github.com/boto/boto/pull/3143

**Comment by aleksandersumowski** [comment]
Author says that the current focus is boto3. Depending on your situation it might be easiest to make the switch ...

https://twitter.com/jsaryer/status/609061783764992000
